src/jest-matchers/index.spec.tsx

```diff
@@ -0,0 +1,70 @@
+const baseUrl = 'http://example.com';
+const param1 = 'param1';
+const param2 = 'param2';
+const testParams = { param1, param2 };
+
+describe('.urlWithTheseParams', () => {
```

Comment:
We should probably have a top-level `describe` block with `__filename`. I am surprised that ESLint did not catch that 🤔 (cf. https://github.com/mozilla/eslint-plugin-amo#describe-with-filename)

Reply:
Note for myself: that's maybe because this file does not have a `module` scope for some reasons, maybe because there is no import/export so its scope is something else...

Reply:
This file is actually causing a problem for the same reason you describe in your second comment. I'm getting the following `tsc` error:

```
src/jest-matchers/index.spec.tsx:1:1 - error TS1208: Cannot compile namespaces when the '--isolatedModules' flag is provided.

1 describe(__filename, () => {
  ~~~~~~~~
```

It seems like this happens because `tsc` thinks this is a global module. I can fix it by changing `isolatedModules` to `false` in `compilerOptions` in `tsconfig.json`, but I'm not sure if we want to do that. It can also be fixed by including an `import` or `export` in the test file, but we don't actually need either of those.

What do you suggest?

Reply:
Note, I found these solutions via https://stackoverflow.com/questions/33977054/cannot-compile-namespaces-when-the-isolatedmodules-flag-is-provided

Reply:
I dealt with this by just adding `export {};` at the bottom of the file.